python/lsst/pipe/tasks/mocks/mockCoadd.py

```diff
@@ -277,7 +277,8 @@ def buildMockCoadd(self, butler, truthCatalog=None, skyMap=None, tract=0):
         tractInfo = skyMap[tract]
         tractWcs = tractInfo.getWcs()
         for patchRef in self.iterPatchRefs(butler, tractInfo):
-            exposure = patchRef.get(self.config.coaddName + "Coadd")
+            # TODO: pybind11 remove `immediate=True` once DM-9112 is resolved
```

Comment:
At risk of being pedantic: there is no actual need to remove `immediate=True` and I suggest alway using it (except for code that actually wants caching, which so far we don't have). That suggest you could delete the comment. On the other hand, I can see an argument for providing a comment for those reading through git history, in which case please consider a slight reword, such as `# immediate=True is required until DM-9112 if fixed`, with or without a `TODO`.

I see at least one other such instance below.